### PR TITLE
Feature/implement end logging 

### PR DESCRIPTION
**Pull request by Unagifan**
Created a basic end logging for all commands

### CODE REVIEW

File: src/xbot/common/controls/sensors/MockXboxController.java [hunk 1]
```diff
@@ -1,14 +1,19 @@
 package xbot.common.controls.sensors;
 
+
 import java.util.HashMap;
 import java.util.Map;
 
+import com.google.inject.Inject;
```
Comment (aschokking) [inline src/xbot/common/controls/sensors/MockXboxController.java]: but why?

File: src/xbot/common/controls/sensors/MockXboxController.java [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+
```
Comment (JohnGilb) [inline src/xbot/common/controls/sensors/MockXboxController.java]: remove blank line

File: src/xbot/common/controls/sensors/MockXboxController.java [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+
 import edu.wpi.first.wpilibj.XboxController;
 import xbot.common.controls.sensors.XboxControllerWpiAdapter.XboxButton;
 import xbot.common.math.XYPair;
 
 public class MockXboxController implements XXboxController {
 
+
```
Comment (JohnGilb) [inline src/xbot/common/controls/sensors/MockXboxController.java]: remove blank line

File: src/xbot/common/controls/sensors/MockXboxController.java [hunk 4]
```diff
@@ -1,14 +1,19 @@
 package xbot.common.controls.sensors;
```
Comment (JohnGilb) [inline src/xbot/common/controls/sensors/MockXboxController.java]: just revert all your changes to this class

File: src/xbot/common/controls/sensors/MockXboxController.java (hunk 1 above)
Comment (Unagifan) [inline src/xbot/common/controls/sensors/MockXboxController.java]: Not me

File: src/xbot/common/controls/sensors/MockXboxController.java (hunk 2 above)
Comment (Unagifan) [inline src/xbot/common/controls/sensors/MockXboxController.java]: ypu

File: src/xbot/common/controls/sensors/MockXboxController.java (hunk 3 above)
Comment (Unagifan) [inline src/xbot/common/controls/sensors/MockXboxController.java]: !

File: src/xbot/common/controls/sensors/MockXboxController.java (hunk 4 above)
Comment (Unagifan) [inline src/xbot/common/controls/sensors/MockXboxController.java]: Okie

**Pull request merged**